vrpn_Imager.h

```diff
@@ -532,7 +532,7 @@ class VRPN_API vrpn_Imager_Region {
                 // varying slowest.  Depth steps are therefore the largest
                 // steps.
                 val =
-                    ((vrpn_uint16 *)
+                    ((const vrpn_uint16 *)
```

Comment:
The strangeness has been there (and worked) since when I wrote it; Matthews improvements won't break anything new.